hydrus/hydraspec/doc_writer.py

```diff
@@ -377,20 +377,7 @@ def generate(self):
                 "label": self.name,
                 "description": "The %s collection" % (self.name,),
                 "domain": "vocab:EntryPoint",
-                "range": "vocab:%sCollection" % (self.name,),
-                "supportedOperation": [
-                    {
-                        "@id": "_:%s_collection_retrieve" % (self.name.lower(),),
-                        "@type": "hydra:Operation",
-                        "method": "GET",
-                        "label": "Retrieves all %s entities" % (self.name,),
-                        "description": None,
-                        "expects": None,
-                        "returns": "vocab:%sCollection" % (self.name,),
-                        "statusCodes": [
-                        ]
-                    }
```

Comment:
That has to be specified in the APIDoc, we cannot decide that. I'll add some things to the writer for checking that.